tests/e2e/env/setup.sh

```diff
@@ -161,6 +161,12 @@ cli wp wc --user=admin tool run install_pages
 echo "Importing some sample data..."
 cli wp import wp-content/plugins/woocommerce/sample-data/sample_products.xml --authors=skip
 
+echo "Adding customer account ..."
+cli wp user create customer customer@woocommercecoree2etestsuite.com --role=customer --user_pass=password
```

Comment:
Can we store the actual values as variables and use them here? We could separately figure out if picking them from config might be feasible later.

Reply:
Sure. We can move this to a variable

Reply:
Moved to variable